topics/imaging/tutorials/tutorial-CP/tutorial.md

```diff
@@ -0,0 +1,633 @@
+>    {: .comment}
+>
+{: .hands_on}
+
+# Object segmentation
+
+## Segment nuclei
+
+Since we are interested in segmenting the nucleoli, you may wonder why we need to segment nuclei first. There are several reasons for that: 
+
+- Get the nuclei features. The intensity, size, shape, number of nucleoli per nucleus, etc. can be informative to study the nucleoli.
+
+- Avoid the detection of wrong spots. In a first segmentation, we detect the nuclei, while in the second pass we will segment the holes (nucleoli). The holes need to fall inside the nuclei, and hence the importance of having them segmented too.
+
+In the first step, we will identify the nuclei that are complete, meaning that they are not touching the borders of the image.
+
+> ### {% icon hands_on %} Hands-on: Segment nuclei that are complete within the boundaries of the image
+>
+> 1. **IdentifyPrimaryObjects** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: `output_pipeline` (output of **Starting Modules** {% icon tool %})
+>    - *"Use advanced settings?"*: `Yes, use advanced settings`
+>        - *"Enter the name of the input image (from NamesAndTypes)"*: `DNA`
+>        - *"Enter the name of the primary objects to be identified"*: `Nuclei`
+>        - *"Typical minimum diameter of objects, in pixel units (Min)"*: `15`
+>        - *"Typical maximum diameter of objects, in pixel units (Max)"*: `200`
+>        - *"Discard objects outside the diameter range?"*: `Yes`
+>        - *"Discard objects touching the border of the image?"*: `Yes`
+>        - *"Threshold strategy"*: `Global`
+>            - *"Thresholding method"*: `Otsu`
+>                - *"Two-class or three-class thresholding?"*: `Two classes`
+>                - *"Threshold correction factor"*: `0.9`
+>        - *"Method to distinguish clumped objects"*: `Shape`
+>            - *"Method to draw dividing lines between clumped objects"*: `Shape`
+>                - *"Automatically calculate size of smoothing filter for declumping?"*: `Yes`
+>                - *"Automatically calculate minimum allowed distance between local maxima?"*: `Yes`
+>        - *"Handling of objects if excessive number of objects identified"*: `Continue`
+>
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > - The name entered to the input image and objects has to be consistent (case sensitive) with the names in *NamesAndTypes* and the tools to follow.
+>    > - The min and max diameter of the objects (`Typical minimum diameter of objects, in pixel units (Min)` and `Typical minimum diameter of objects, in pixel units (Max)`) will have to be adjusted to the resolution of the images.
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> We are using here Otsu's method for segmentation. What other segmentation options are available? What is the difference between them?
```

Comment:
Maybe mention "Otsu thresholding method" (add thresholding) and then in the answer mention for that it is for the "Adaptative Threshold strategy").

Reply:
Not sure if I fully get your comment here :) I've added thresholding to the sentence, was there anything else to change?